scripts/install/install_macos.sh

```diff
@@ -161,6 +163,32 @@ Usage:
 
   $(fg_yellow '-u, --upgrade')
       Stops the collector services and upgrades the collector via brew.
+  
+  $(fg_yellow '-e, --endpoint')
+      Defines the endpoint of an OpAmp compatible management server for this collector install.
+      
+      Specifying this will install the collector in a managed mode, as opposed to the
+      normal headless mode.
+  
+  $(fg_yellow '-e, --endpoint')
```

Comment:
@BinaryFissionGames is this supposed to be in here twice?

Reply:
It was not supposed to be there twice, good catch!